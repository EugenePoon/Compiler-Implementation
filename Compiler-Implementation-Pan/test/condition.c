int main()
{
    int c = 10;
    int b = 8;
    int a = c>b ? c:b;
    print a;
}