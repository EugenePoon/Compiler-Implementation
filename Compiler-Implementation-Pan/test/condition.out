24 19 0 5 25 13 0 10 12 13 0 1 1 0 8 12 13 0 2 1 13 11 13 0 1 1 11 10 7 17 35 13 11 16 40 13 0 1 1 11 12 13 0 2 1 11 22 21 3